<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCPWidget</class>
 <widget class="QWidget" name="GCPWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>163</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GCPWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gridGroupBox">
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="title">
      <string>Grid</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="grid"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="coordinatePlaneGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedKingdom"/>
     </property>
     <property name="title">
      <string>Coordinate Plane</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="coordinatePlane"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
